src/components/Button/__stories__/index.stories.tsx

```diff
@@ -141,23 +141,17 @@ Action.parameters = {
 }
 Action.decorators = [
   () => (
-    <>
+    <div style={{ display: 'flex', flexWrap: 'wrap', gap: '16px' }}>
```

Comment:
I think this is happening a few times in scaleway-ui, shouldn't we replace all mr/mb with a flex gap on the parent? (in a separate PR)

Reply:
Yes absolutely, we are in the process of removing xstyled props, so whenever you can please do :)